src/nvme/types.h

```diff
@@ -1114,6 +1114,25 @@ struct nvme_id_ctrl {
 	__u8			vs[1024];
 };
 
+enum nvme_cmic {
```

Comment:
I  think it  would be good to add documentation to all the new types you are adding. IIRC, I forgot to ask for this in the last few PRs. It's not a real blocker, though please consider to provide the documentation update in future. Thanks.

Reply:
Sure of course will to it later. Thank you.